static/src/javascripts/projects/common/modules/experiments/tests/rtrt-email-message.js

```diff
@@ -76,18 +79,22 @@ define([
             {
                 id: 'targeted-loyal-A',
                 test: function () {
-                    createMessage('p2lq8cs6r'); // 10 visits or more to the Guardian
+                    kruxSegmentId = 'p2lq8cs6r';
```

Comment:
I think it'd be better to keep the segment id as a parameter to `createMessage`. Maybe it should return a function so you can just pass `createMessage(id)` to mediator.